defaults/main.yml

```diff
@@ -9,3 +9,9 @@ dell_powerconnect_switch_config: []
 # dicts. Each dict contains a 'description' (or 'name' in case of vlans) item and 
 # a 'config' item which should contain a list of per-interface configuration.
 dell_powerconnect_switch_interface_config: {}
+
+# When true, switch configuration will be written to the startup config file.
+write_memory: false
```

Comment:
Could you add a `dell_powerconnect_switch_` prefix to these? It's the closest thing we have to namespacing in Ansible.

Reply:
I've added a prefix in the latest commit.